manifests/params.pp

```diff
@@ -8,6 +8,18 @@
     fail('The puppet_agent class requires stringify_facts to be disabled')
   }
 
+  # Which services should be started after the upgrade process?
```

Comment:
@ekinanp were your objections here about the comment or the behavior? If it was the comment, I've updated it. If it was behavior, I'm not sure I see how this is changing the behavior from how it used to be (outside of removing mcollective for puppet6), but this is the first time I've made changes to a module that are more than a few lines -- I don't know what I'm missing.

Reply:
Just the comment